graylog2-server/src/main/resources/log4j2.xml

```diff
@@ -11,7 +11,7 @@
     <Loggers>
         <!-- Application Loggers -->
         <Logger name="org.graylog2" level="info"/>
-        <Logger name="com.github.joschi.jadconfig" level="warn"/>
+        <Logger name="org.graylog.jadconfig" level="warn"/>
```

Comment:
The library's package name is still `com.github.joschi`. So this change is not needed.

Reply:
Fixed, sorry for the confusion!